docs/install.rst

```diff
@@ -161,9 +161,9 @@ using ``zsh``, the ``pyenv`` docs have other routes ::
 
     $ brew install pyenv-virtualenv
 
-2. Configure your shell to init ``pyenv-virtualenv`` on start - again, this is noted in the ``pyenv-virtualenv`` project's `own docs <https://github.com/pyenv/pyenv-virtualenv>`_, in more detail ::
+2. Configure your shell to init ``pyenv-virtualenv`` on start - again, this is noted in the ``pyenv-virtualenv`` project's `own documentation <https://github.com/pyenv/pyenv-virtualenv>`_, in more detail. The following will slot in a command that will work as long as you have pyenv-virtualenv installed::
 
-    $ eval "$(pyenv virtualenv-init -)"
+    $ echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.zshrc
```

Comment:
Do we assume somewhere that all users are using ZSH?

Reply:
yep, there's some text earlier in the page about the assumptions and how to find instructions for other setups:

> The following assumes you are on MacOS, using zsh as your shell and [Homebrew](https://brew.sh/) as your package manager. If you are not, there are installation instructions for a variety of platforms and shells in the READMEs for the two pyenv projects.